roles/spark/tasks/main.yml

```diff
@@ -148,6 +151,34 @@
             dest="{{ spark_usr_parent_dir }}/spark-{{ spark_version }}/conf/hive-site.xml"
   tags: ["config"]
 
+#Install Numpy & SciPy and its dependencies
+- name: Install pip packages
+  pip: name={{ item }} executable=pip3 extra_args=--user
+  with_items:
+    - numpy
+    - scipy
+    - matplotlib
+    - ipython
+    - jupyter
+    - pandas
+    - sympy
+    - nose
+
+#Install and enable ATLAS and OpenBLAS for netlib-java to be used by Breeze
+#Breeze is a library for linear algebra operations used by SparkML
+- name: Install Atlas and OpenBLAS packages
+  apt: name={{ item }}
+  with_items:
+    - libatlas3-base
+    - libopenblas-base
+- name: Configure Atlas and OpenBLAS
+  shell: "{{ item }}"
+  with_items:
+    - sudo update-alternatives --config libblas.so
```

Comment:
When I run this on my workstation I get
```
update-alternatives: error: no alternatives for libblas.so
```
If there where multiple alternatives it would interactivly ask which blas you want to make the default, so this command should ever work like this.

 `update-alternatives` is only needed when you have more than one libblas installed.
So I would remove this task.